storage/blob.go

```diff
@@ -649,7 +649,14 @@ func (b BlobStorageClient) CreateBlockBlobFromReader(container, name string, siz
 //
 // See https://msdn.microsoft.com/en-us/library/azure/dd135726.aspx
 func (b BlobStorageClient) PutBlock(container, name, blockID string, chunk []byte) error {
-	return b.PutBlockWithLength(container, name, blockID, uint64(len(chunk)), bytes.NewReader(chunk))
+	return b.PutBlockWithLength(container, name, blockID, uint64(len(chunk)), bytes.NewReader(chunk), nil)
+}
+
+func (b BlobStorageClient) PutBlockWithContentType(container, name, blockID string, chunk []byte, contentType string) error {
```

Comment:
I think this is pretty trivial to achieve with the method below. Let's remove this method as achieving this is as simple as passing a simple map initializer value to the other method. Also the CI build fails because there's no godoc comment on this.